editor.planx.uk/src/@planx/components/List/model.ts

```diff
@@ -20,3 +21,9 @@ export const parseContent = (data: Record<string, any> | undefined): List => ({
   schema: cloneDeep(data?.schema) || SCHEMAS[0].schema,
   ...parseMoreInformation(data),
 });
+
+export const validationSchema = object({
+  schema: object({
+    max: number().optional().min(2, "Please use a Page component - the maximum must be greater than 1 item"),
```

Comment:
Just thinking about it, is this error intended for when the Schema is exposed in the Editor, because right now we're defining what you can select here?

If I was thinking as a user, I'd feel a bit put out if I selected an option and it told me I was wrong / can't have that option

Reply:
Thanks - very good observation! Message updated ✅ 

This error will no longer be visible once we remove Advert Consent from the List schema directory. From that point onwards this message will only be viewable if an Editor (with access to GitHub) creates a new schema with `max: 1` set. So although it's a poor UI, it's incredibly edge casey and could only possibly be seen by the internal content team.

Longer term, we'll have a UI to generate schemas. At this point, we'll have a `max` field with this validation control baked in to avoid this possibility.